src/js/modules/core/components/MastheadBar.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import injectSheet from 'react-jss';
+import { Link } from 'react-router-dom';
+
+import NavButton from './NavButton';
+
+const styles = {
+  MastheadBar: {
+    backgroundColor: '#fff',
+    height: '40px',
+    padding: '0px 60px',
+    position: 'fixed',
+    width: '100%',
+    top: '0px',
+  },
+  barContainer: {
+    borderBottom: '1px solid #ddd',
+    height: '100%',
+    margin: '0 auto',
+    position: 'relative',
+    width: '100%',
+    textAlign: 'center',
+  },
+  quickNav: {
+    float: 'left',
+    marginTop: '9px',
+    '& button': {
+      marginRight: '24px',
+    },
+  },
+  brandingLink: {
+    color: '#000',
+    fontFamily: 'Old English Text MT',
+    fontSize: '26px',
+    marginTop: '4px',
+    textDecoration: 'none',
+  },
+  userTools: {
+    float: 'right',
+    marginTop: '9px',
+    '& button': {
+      marginLeft: '24px',
+    }
+  }
+};
+
+const MastheadBar = ({ classes }) => {
+  return (
+    <div className={classes.MastheadBar}>
+      <div className={classes.barContainer}>
+        <div className={classes.quickNav}>
+          <NavButton buttonType="sections" className={classes.leftEdgeNavButton}/>
```

Comment:
this `buttonType` prop determines what icon to show. For example, the section NavButton needs a hamburger menu. So this isn't really a label, it determines the type of icon shown too.